youtube_dl/extractor/peertube.py

```diff
@@ -410,24 +410,24 @@ class PeerTubeIE(InfoExtractor):
                             video\.colibris-outilslibres\.org|
                             tube\.svnet\.fr|
                             peertube\.video|
-                            peertube3\.cpy\.re|
                             peertube2\.cpy\.re|
+                            peertube3\.cpy\.re|
                             videos\.tcit\.fr|
                             peertube\.cpy\.re|
                             canard\.tube
                         )'''
-    _UUID_RE = r'[\da-fA-F]{8}-[\da-fA-F]{4}-[\da-fA-F]{4}-[\da-fA-F]{4}-[\da-fA-F]{12}'
+    _UUID_RE = r'[\da-zA-Z]{22}|[\da-fA-F]{8}-[\da-fA-F]{4}-[\da-fA-F]{4}-[\da-fA-F]{4}-[\da-fA-F]{12}'
```

Comment:
Wouldn't this break the old UUID's, assuming old ones are still in use ?

Reply:
Hi,

No, the regex should be able to detect new "short uuids" (22 characters) **or** classic uuids

To prevent potential regression, I added tests that mix the new/old URLs:
 * https://github.com/ytdl-org/youtube-dl/pull/29475/files#diff-2efe313d1198773f8eeb5db14ac89d57b93837515e2bc0e141eed8856ab133deR455
 * https://github.com/ytdl-org/youtube-dl/pull/29475/files#diff-2efe313d1198773f8eeb5db14ac89d57b93837515e2bc0e141eed8856ab133deR466